src/UI/UIView.cpp

```diff
@@ -915,9 +933,10 @@ void UIView::draw_ui_callback()
 
 void UIView::init_ui(bool is2D, bool lookingGlass)
 {
-
+  
   if (!m_initialized)
-  {
+  { 
+    
 #ifdef WITH_NRRD
     fileDialog.SetTypeFilters({".txt", ".nrrd"});
 #elseif
```

Comment:
Spacing looks pretty weird here, I'd definitely run the program through the linter again

Reply:
I reformatted the file. That's what it gives.